app/policies/roles/site/institution.json.erb

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "name": "Site <%=  %> Admin",
+    "policy": {
+      "statement": [
+        {
+          "resource": "site/<%= site_id %>",
+          "action": [
+            "site:read",
+            "site:update",
+            "site:delete"
+          ],
+          "delegable": true
+        }
+      ]
+    }
+  },
+  {
+    "name": "Site <%=  %> Reader",
+    "policy": {
+      "statement": [
+        {
+          "resource": "site/<%= site_id %>",
+          "action": [
+            "site:read",
+            "site:update",
+            "site:delete"
+          ],
+          "delegable": true
+        }
+      ]
+    }
+  },
+  {
+    "name": "Site <%=  %> Technician",
+    "policy": {
+      "statement": [
+        {
+          "resource": "site/<%= site_id %>",
+          "action": [
+            "site:read",
+            "site:update",
+            "site:delete"
+          ],
+          "delegable": true
+        }
+      ]
+    }
+  },
+  {
+    "name": "Site <%=  %> Supervisor",
+    "policy": {
+      "statement": [
+        {
+          "resource": "site/<%= site_id %>",
+          "action": [
+            "site:read",
+            "site:update",
+            "site:delete"
+          ],
```

Comment:
According to case #591, the supervisor can publish diagnoses. Since we don't have that action yet, I'd suggest removing this role for now.